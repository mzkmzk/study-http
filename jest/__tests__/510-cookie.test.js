const curl = require('../curl/index')
/**
 * 考点 后端服务器如何设置cookie
 */
it('服务器如何设置cookie', async function(){
  const { status, body, row, stdout, stderr } = await curl(`curl -vs -H 'Host: study-http.404mzk.com' http://47.115.81.18:9500/set-cookie`)
  expect(row).toMatchSnapshot()
  
  //expect(stdout + stderr).toMatchSnapshot()
})