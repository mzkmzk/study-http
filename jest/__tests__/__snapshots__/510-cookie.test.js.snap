// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`服务器如何设置cookie 1`] = `
"GET /set-cookie HTTP/1.1
User-Agent: curl/7.38.0
Accept: */*
Host: study-http.404mzk.com

HTTP/1.1 200 OK
Server: nginx/1.17.10
Date: mock-time
Content-Type: application/octet-stream
Content-Length: 22
Connection: keep-alive
Set-Cookie: cookie-key=cookie-value; Domain=404mzk.com; Path=/; Max-Age=1000;

hello-world set-cookie"
`;
